frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -507,6 +507,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
       id="dropdown-selectbox"
       selectedKey={selectedTargetNamespace}
       onChange={setTargetNamespace}
```

Comment:
We need to handle this onChange prop in the NsDropdown component. We added our own onChange function to that component and no longer pass this down to the ListDropdown via the spread operator.

Reply:
The `onChange()` handler in `NSDropdown` component now calls `props.onChange()`

Reply:
This also needs to be updated to prevent a React warning. The call signature of `NsDropdown`'s `props.onChange` function accepts three args. If you pass more than one arg to the `React.useState` setter callback, it will trigger a console warning.  
```suggestion
      onChange={(ns) => setTargetNamespace(ns)}
```